test/parallel/test-promises-warning-on-unhandled-rejection.js

```diff
@@ -0,0 +1,31 @@
+// Flags: --no-warnings
```

Comment:
@jasnell It should never reach stderr since we are listening to it. is there a bug?